.gitignore

```diff
@@ -4,3 +4,5 @@ dist
 .tox
 _build
 *.egg-info/
+.coverage
+.python-version
```

Comment:
I'm trying to decide whether it's better to exclude .python-version from git in this repo. On the one hand, now that I'm using pyenv, it'd be handy to have the version specified in the repo in a way that would automatically get me the right version. On the other hand, what's the "right version" for this package that can run on multiple versions? It's probably simplest to just exclude it, at least for this repo. Thoughts?

Reply:
I've gone back and forth about this and I don't have a good answer yet. I'm actually thinking that maybe we should include a `.python-version` for the reason that you mention. This is the `.python-version` that I have for this project:
```
3.6.10
3.7.6
3.8.1
3.9.0
```
Without that, tox doesn't work locally for me since I don't otherwise have the right python versions accessible to tox.

So I think I'll revert this change and include the file in the repo.

Reply:
That makes sense. I haven't yet internalized that we can put multiple versions into .python-version, and that's perfect for this.